src/modules/sensors/parameters.cpp

```diff
@@ -188,6 +188,9 @@ void initialize_parameter_handles(ParameterHandles &parameter_handles)
 	(void)param_find("SYS_CAL_TDEL");
 	(void)param_find("SYS_CAL_TMAX");
 	(void)param_find("SYS_CAL_TMIN");
+
+	(void)param_find("SYS_HAS_MAG");
```

Comment:
This isn't necessary.

Reply:
i think i invalidated @bkueng's review/approval by squashing in his requested changes. i'll remove this line. @dagar -- want me to squash that in? separate commit or fix later?

Reply:
never mind, i just pushed it separately since it's not in my original commit anyway

Reply:
It's pretty harmless to leave if you aren't already rebasing.